sample/src/main/java/com/chuckerteam/chucker/sample/HttpBinClient.kt

```diff
@@ -120,4 +124,19 @@ class HttpBinClient(
             }
         })
     }
+
+    private fun getResponsePartially() {
```

Comment:
nit: I think we should consider extracting this to a separate class.
We have this class called `HttpBinClient` but starts to have a lot of sample HTTP calls to other endpoints.

Reply:
Yes, you're right. It makes sense to not evolve this into a ball of mud. I, or any other volunteer, can make it in this PR or a follow-up one. I agree that it is not crucial to do it right now.

Reply:
I support the idea of doing such change in another PR.